src/app/inventory/dim-item-info.ts

```diff
@@ -57,6 +57,13 @@ export const tagConfig = {
 
 export type TagValue = keyof typeof tagConfig | 'clear' | 'lock' | 'unlock';
 
+/**
+ * Helper function to check if a string is TagValue type and declare it as one.
+ */
+export function isTagValue(value: string): value is TagValue {
+  return [...Object.keys(tagConfig), 'clear', 'lock', 'unlock'].includes(value);
```

Comment:
Since `tagConfig` is constant, it's best to generate this list outside of the function so it isn't rebuilt on every call.

Reply:
Sure, done